pages/login.js

```diff
@@ -0,0 +1,22 @@
+import Image from 'next/image';
+
+import Background from '../components/Background';
+import LoginForm from '../components/forms/Login';
+
+import logo from '../public/logo.png';
+
+const Login = () => {
+  return (
+    <Background imagePath="/background.png">
+      <div className="mb-6">
```

Comment:
Should we change this to `mb-4` as the space between the heading and the form is `32px` but currently there's `16px` on the text and `24px` on this wrapper?

The current style | The design
--- | ---
![Screen Shot 2565-03-28 at 10 58 08](https://user-images.githubusercontent.com/1772999/160324299-c45b6a5d-cb96-4b71-a33b-6fb6d2eb7627.png) ![Screen Shot 2565-03-28 at 10 57 57](https://user-images.githubusercontent.com/1772999/160324291-c6d7000f-d3cb-41d4-b171-71228932953c.png) | ![Screen Shot 2565-03-28 at 10 58 44](https://user-images.githubusercontent.com/1772999/160324325-b46d1675-2490-4778-b28b-38c5659a984a.png)

Reply:
sure, done in https://github.com/tyrro/react-survey/pull/50/commits/42a749da2be9185adad4215698e02f740cb23b2f